Makefile

```diff
@@ -305,6 +305,7 @@ rpkg:
 	Rscript -e "require(mxnet); mxnet:::mxnet.export(\"R-package\")"
 	Rscript -e "require(roxygen2); roxygen2::roxygenise(\"R-package\")"
 	R CMD build --no-build-vignettes R-package
+	mv mxnet_*.tar.gz mxnet_current_r.tar.gz
```

Comment:
the docs says:
These commands create the MXNet R package as a tar.gz file that you can install as an R package. To install the R package, run the following command, use your MXNet version number:

    R CMD INSTALL mxnet_0.7.tar.gz